examples/Main.ci

```diff
@@ -12,6 +12,15 @@ mainmodule Main {
 
     initnode void initialize(void);
 
+    include "PerLeafFunctions.h";
+
+    PUPable FirstCollisionFn;
```

Comment:
I genuinely have no clue what to call these, so feedback is welcome.

Reply:
For SPH:
FirstSphFn => DensityFn
SecondSphFn => ForceFn
ThirdSphFn => SymmetricForceFn or "ReactionFn"
For collisions:
FirstCollisionFn => CollisionResolveFn
SecondCollisionFn => CropFn